pkg/ddevapp/drupal.go

```diff
@@ -93,6 +93,18 @@ $settings['file_scan_ignore_directories'] = [
 // This will prevent Drupal from setting read-only permissions on sites/default.
 $settings['skip_permissions_hardening'] = TRUE;
 
+// This will ensure the site can only be accessed through the intended host names.
+// Additional host patterns can be added for custom configurations.
+$settings['trusted_host_patterns'] = ['.*'];
+
+// This specifies the default configuration sync directory.
+if (empty($config_directories[CONFIG_SYNC_DIRECTORY])) {
+  $config_directories[CONFIG_SYNC_DIRECTORY] = 'sites/default/files/sync';
+  if (!file_exists($config_directories[CONFIG_SYNC_DIRECTORY])) {
+    mkdir($config_directories[CONFIG_SYNC_DIRECTORY]);
```

Comment:
Oh, I was thinking we should do the mkdir in go, interesting. I wonder where it belongs. in php though it probably needs 'recursive' set in case the dir doesn't exist yet, http://php.net/manual/en/function.mkdir.php